### DISCUSSION TITLE
Macros in syslink.h file

### DISCUSSION THREAD

**Discussion by b0bbarker** [bitcraze/discussions#1504] [Q&A] [answered]
Hello everyone,

recently I'm trying to understand the example app "app_peer_to_peer" for which I want to develop a unicast model afterwards. However as I was trying some new things on the code I came across the following thing:

```
#define SYSLINK_RADIO_P2P           0x08
#define SYSLINK_RADIO_P2P_ACK       0x09
#define SYSLINK_RADIO_P2P_BROADCAST 0x0A
```

The macros for the SYSLINK_RADIO have numbers for each subelement from 0x00 to 0x0A. So I took the SYLINK_RADIO_P2P_BROADCAST macro and renamed it from 0x0A to 0x0B and tried to run the same code. The code is fresh from github so the only thing modified was that part. Even so the broadcast packets weren't either recognized or sent. 

I also tried to define my own makros, give them an own mask and change the functions 'syslinkRouteIncomingPacket' and 'radiolinkSendP2PPacketBroadcast' only at those places in which thr packettype of the syslink packets is used, which also didn't work.

So my question is, what could be the reason for this error? Is there a place deeper in the code where the Sylinkpackettype is asked again? Or maybe the console just doesn't print the packets?

**Comment by gemenerik** [comment, marked as answer]
These are [syslink](https://www.bitcraze.io/documentation/repository/crazyflie2-nrf-firmware/master/protocols/syslink/) defines. Syslink is the protocol used between the STM32 and NRF51. The defines [in the STM firmware](https://github.com/bitcraze/crazyflie-firmware/blob/master/src/hal/interface/syslink.h#L52) should match those [in the NRF firmware ](https://github.com/bitcraze/crazyflie2-nrf-firmware/blob/master/interface/syslink.h#L93) so they have a unified understanding of the package type.

**Reply by b0bbarker** [reply]
Now that makes sense. Thank you very much. I really looked a lot through all of it but I never would've looked into the other firmware. Thanks a lot
